installer/release/bootstrap-install-script.sh.template

```diff
@@ -223,7 +230,8 @@ else
 fi
 
 echo "$CLOUDFLOW_INSTALLER_YAML" | \
-  sed -e "s/CLOUDFLOW_INSTALLER_IMAGE_TAG_PH/$CLOUDFLOW_INSTALLER_IMAGE_TAG/" | \
+  sed -e "s~CLOUDFLOW_INSTALLER_IMAGE_TAG_PH~$CLOUDFLOW_INSTALLER_IMAGE_TAG~" \
+      -e "s~CLOUDFLOW_INSTALLER_IMAGE_PH~$CLOUDFLOW_INSTALLER_IMAGE~" | \
```

Comment:
Because `$CLOUDFLOW_INSTALLER_IMAGE` likely contains slashes, to avoid confusing `sed`, a new separator is needed.